install/index.md

```diff
@@ -17,58 +17,70 @@ redirect_from:
 - /engine/installation/
 - /en/latest/installation/
 - /linux/
+toc_max: 2
 ---
 
-Docker Engine - Community is ideal for developers and small
-teams looking to get started with Docker and experimenting with container-based
-apps. Docker Engine - Community has three types of update channels, **stable**, **test**, and **nightly**:
+Docker Engine is an open source containerization technology for building and
+containerizing your applications. Docker Engine acts as a client-server
+application with:
+* A server with a long-running daemon process ([`dockerd`](https://docs.docker.com/engine/reference/commandline/dockerd/)).
+* APIs which specify interfaces that programs can use to talk to and
+  instruct the Docker daemon.
+* A command line interface (CLI) client ([`docker`](https://docs.docker.com/engine/reference/commandline/cli/)).
```

Comment:
just now noticed this; should these be relative URLs (without `https://docs.docker.com/`)?

Reply:
Fixed via the latest commit